types/chrome/index.d.ts

```diff
@@ -774,7 +774,7 @@ declare namespace chrome.browserAction {
 
     export interface BadgeTextDetails {
         /** Any number of characters can be passed, but only about four can fit in the space. */
-        text?: string | undefined;
+        text?: string | null;
```

Comment:
I only glanced at this because it was at the head but doesn't this change regress #54317?  Should it be `string | null | undefined`?

@sandersn This is probably not the first regression of your widespread changes.

Reply:
the `text?` means that undefined is a valid type. If you view the changes in the playground, text is allowed to be undefined.

Reply:
we do need to fix that regression though, I'll submit a PR, but it'll take a while to get through, since it will have to go through the maintainer queue.

Reply:
This is not specified as a field that can be set to `undefined` in the manuals, so I removed it.
(although `undefined` would probably work anyway...)

Reply:
That PR, #57471, is now merged, so we're all good!